style.css

```diff
@@ -92,11 +96,199 @@ body{
 .collab-btn:active{
     border: 2px solid var(--primary-color);
 }
+/* END HEADLINE SECTION */
+/* START WORK SECTION */
+.works{
+    margin-bottom:20px;
+}
+.title h1{ 
+    color:var(--primary-color);
+    font-family: 'Playfair Display', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 55px;
+    line-height: 100%;
+    margin:0 0 35px 20px;
+}
+.project-cards{
+    background-color:var(--primary-color);
+    display:flex;
```

Comment:
I suggest creating a separate class that makes use of this `display: flex` property so you can truly reuse it without always repeating this rule again.

Like this...
```

.flex {
  display: flex;
}
```

You can then apply this as needed in the container as a class.